lib/_stream_transform.js

```diff
@@ -217,10 +218,10 @@ function done(stream, er, data) {
   var ts = stream._transformState;
 
   if (ws.length)
-    throw new Error('Calling transform done when ws.length != 0');
+    throw new errors.Error('ERR_CALLING_WS_LENGTH');
```

Comment:
I'd agree a better name for the error would be good.  Maybe 'ERR_TRANSFORM_WITH_LENGTH_0'.